packages/dcos-integration-test/extra/test_service_discovery.py

```diff
@@ -85,7 +87,7 @@ def _service_discovery_test(dcos_api_session, docker_network_bridge):
                         stop_max_delay=DNS_ENTRY_UPDATE_TIMEOUT * 1000,
                         retry_on_result=lambda ret: ret is None,
                         retry_on_exception=lambda x: False)
-        def _pool_for_mesos_dns():
+        def _pool_for_mesos_dns() -> Any:
```

Comment:
It's not returning just any. It seems to be `Union[None, dict]`.

Reply:
I prefer to use Optional as `Optional[...]` is a shorthand notation for `Union[..., None]`, thanks for help will do this